certified-connectors/Expensya/apiDefinition.swagger.json

```diff
@@ -0,0 +1,10032 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Expensya Management",
+    "version": "1.0",
+    "description": "Expensya Account Management",
```

Comment:
Could the description please be expanded to discuss the main purpose and value that the connector offers? Maximum character count for this is 500 characters, for reference.